src/tools/profile/read_profile.c

```diff
@@ -558,7 +558,7 @@ static void show_profile_data_log(profile_data_t *data, options_t *opts,
                     reqid = reqid_ctr++;
                     kh_value(&reqids, hash_it) = reqid;
                 }
-                action = "NEW";
+                action = "NEW ";
```

Comment:
then better also add whitespace for `action = "FREE";` below

Reply:
it's not needed after free because the "name" is empty string in case of free
PSB the log with this PR applied
```
     0.961  ucp_tag_recv_nbx 1.102 {                              tag_recv.c:208  ucp_tag_recv_nbx()
     0.266      NEW ucp_request {2}                               tag_recv.c:224  ucp_tag_recv_nbx_inner()
     0.051      ucs_memtype_cache_lookup 0.633 {             memtype_cache.c:279  ucs_memtype_cache_lookup()
     0.056          ucs_pgtable_lookup 0.515 {               memtype_cache.c:289  ucs_memtype_cache_lookup_inner()
     0.515          }
     0.062      }
     0.151  }
     0.170  ucp_tag_send_nbx 2.596 {                              tag_send.c:229  ucp_tag_send_nbx()
     0.019      ucp_tag_send_inline 0.041 {                       tag_send.c:252  ucp_tag_send_nbx_inner()
     0.041      }
     0.057      ucs_memtype_cache_lookup 0.135 {             memtype_cache.c:279  ucs_memtype_cache_lookup()
     0.044          ucs_pgtable_lookup 0.058 {               memtype_cache.c:289  ucs_memtype_cache_lookup_inner()
     0.058          }
     0.033      }
     0.125      start_contig_short {0}                         ucp_request.c:600  ucp_request_send_start()
     2.077      complete_send {0}                            ucp_request.inl:212  ucp_request_complete_send()
     0.141  }
     0.426  ucp_eager_only_handler 0.408 {                       eager_rcv.c:133  ucp_eager_only_handler()
     0.035      eager_recv {2}                                   eager_rcv.c:26   ucp_eager_expected_handler()
     0.172      memcpy_recv 0.064 {                          ucp_request.inl:588  ucp_request_unpack_contig()
     0.064      }
     0.022      complete_recv {2}                            ucp_request.inl:224  ucp_request_complete_tag_recv()
     0.115  }
     0.327  ucp_request_free 0.145 {                           ucp_request.c:237  ucp_request_free()
     0.074      FREE {2}                                       ucp_request.c:223  ucp_request_release_common()
     0.070  }
```

Reply:
ok, ic